intro.Rmd

```diff
@@ -38,7 +38,11 @@ This philosophy is realised primarily through the devtools package, which is the
 
 As always, the goal of devtools is to make package development as painless as possible. It encapsulates the best practices developed by first author Hadley Wickham, initially from years as a prolific solo developer. More recently, he has assembled a team of ~10 developers at RStudio, who collectively look after ~150 open source R packages, including those known as [the tidyverse](https://www.tidyverse.org). The reach of this team allows us to explore the space of all possible mistakes at an extraordinary scale. Fortunately, it also affords us the opportunity to reflect on both the successes and failures, in the company of expert and sympathetic colleagues. We try to develop practices that make life more enjoyable for both the maintainer and users of a package. The devtools meta-package is where these lessons are made concrete.
 
-Devtools works hand-in-hand with RStudio, which I believe is the best development environment for most R users. The only real competitor is [ESS](http://ess.r-project.org/), emacs speaks statistics, which is a rewarding environment if you're willing to put in the time to learn emacs and customise it to your needs. The history of ESS stretches back over 20 years (predating R!), but it's still actively developed and many of the workflows described in this book are also available there.
+Devtools works hand-in-hand with RStudio, which we believe is the best development environment for most R users. The main alternative is [Emacs Speaks Statistics](http://ess.r-project.org/) (ESS), which is a rewarding environment if you're willing to put in the time to learn Emacs and customise it to your needs. The history of ESS stretches back over 20 years (predating R!), but it's still actively developed and many of the workflows described in this book are also available there.
```

Comment:
It might be worth mentioning vim and [Nvim-R](https://github.com/jalvesaq/Nvim-R) as well here? I use it and I personally know a few other people in the R community that do, so it is not completely obscure...

ESS and emacs definitely has more history with S and R, but Vim is much more widely used in the general software developer community than emacs. The latest SO survey had 25% of developers using Vim vs only 4.5% using emacs (https://insights.stackoverflow.com/survey/2019#development-environments-and-tools)

On the other hand RStudio only had 3.4% users from that survey, so clearly the majority of the SO respondents were not R users.